src/groups/special_orthogonal.jl

```diff
@@ -44,6 +44,13 @@ function allocate_result(
     return allocate(q)
 end
 
+function exp_lie!(::SpecialOrthogonal{2}, q, X)
```

Comment:
I think we usually have 3 arguments here? `exp!(M, q, p, X)` in order to have a different result variable (`q`) compared to the start `p`? Or did I mix this up and `pis the identity here anyways?

Reply:
In `lie` variants of methods `p` is the identity. `exp_lie!(M, q, X)` is like `exp!(M, q, identity_element(M), X)`.

Reply:
Ok, thanks for the reminder :)